src/test/spec/sessions/operation.rs

```diff
@@ -0,0 +1,1178 @@
+use std::{collections::HashMap, fmt::Debug, ops::Deref};
```

Comment:
This is all quite similar to the operations stuff in the other runners, just some minor adjustments to fit in with the specific test schema for sessions/transactions.

Reply:
The README for the runner claims it has the same format as the transactions runner, which should mean it has the v2 format (transactions / retryable reads / crud v2). Is there any chance we'll be able to reuse code there instead of introducing an entirely new runner?